File: core/src/com/shinkson47/game/input/KeyHandler.java
package com.shinkson47.game.input;

import com.badlogic.gdx.Gdx;
import com.badlogic.gdx.Input;
import com.shinkson47.game.rendering.Renderer;
import com.shinkson47.game.world.World;

/**
 * <h1></h1>
 * <br>
 * <p>
 *
 * </p>
 *
 * @author <a href="https://www.shinkson47.in">Jordan T. Gray on 12/04/2021</a>
 * @version 1
 * @since v1
 */
public class KeyHandler {
    // TODO dynamic bindings (Key id > consumer kinda deal)

    public static void Poll(){
        if(Gdx.input.isKeyPressed(Input.Keys.ESCAPE))
                Gdx.app.exit();

        if (Gdx.input.isKeyJustPressed(Input.Keys.F5)) {
            World.create();
            Renderer.create();
        }


        if (Gdx.input.isKeyJustPressed(Input.Keys.F6))
            World.focusedWorld.swapTiledInterp();
    }
}

File: core/src/com/shinkson47/SplashX6/input/keys/KeyHandler.kt
package com.shinkson47.SplashX6.input.keys

import com.badlogic.gdx.Gdx
import com.badlogic.gdx.Input
import com.badlogic.gdx.InputMultiplexer
import com.shinkson47.SplashX6.game.GameHypervisor

/**
 * #
 * @author [Jordan T. Gray on 12/04/2021](https://www.shinkson47.in)
 * @version 1
 * @since v1
 */
object KeyHandler {

    @Deprecated("See KeyBinder.")
    fun Poll() {
        if (! GameHypervisor.inGame) return // after here requires in game
            GameHypervisor.gameRenderer?.cam!!.boost(Gdx.input.isKeyPressed(Input.Keys.SHIFT_LEFT))
    }
}